src/commons/TestRailApi.ts

```diff
@@ -0,0 +1,67 @@
+import axios, { AxiosBasicCredentials, AxiosPromise, AxiosRequestConfig } from 'axios';
+
+const BASE_URL: string = 'https://cloudinary.testrail.net/index.php?/api/v2/';
+
+enum ApiMethods {
+  updateCase = 'update_case/',
+}
+
+export enum AutomationFieldOptions {
+  automated = 3,
+  applitoolsCandidate = 4,
+}
+
+/**
+ * TestRail api
+ */
+export class TestRailApi {
+  private static instance: TestRailApi;
+  private basicAuth: AxiosBasicCredentials;
+
+  public static get Instance(): TestRailApi {
+    if (!this.instance) {
+      this.instance = new TestRailApi();
+    }
+
+    return this.instance;
+  }
+
+  /**
+   *   Change test 'Automation' field
+   *   @param testID test id as appear on testRail site
+   *   @param value choose one of AutomationOptions
+   */
+  public changeTestAutomationField(testID: string, value: AutomationFieldOptions): AxiosPromise {
```

Comment:
it need to be more generic like "updateFieldInTestRails" and it should get
`testId:string, {key, value}`